src/DiskIO/DiskThreads/DiskThreadsDiskFile.cc

```diff
@@ -189,7 +189,7 @@ DiskThreadsDiskFile::close()
         ioRequestor->closeCompleted();
         return;
     } else {
-        debugs(79, DBG_CRITICAL, HERE << "DiskThreadsDiskFile::close: " <<
+        debugs(79, DBG_CRITICAL, "DiskThreadsDiskFile::close: " <<
```

Comment:
Same answer: This PR automatically removes HERE. Nothing else should be required.